set.mm

```diff
@@ -749468,4 +749469,325 @@ to Davis and Putnam (1960).  (Contributed by David A. Wheeler,
 
 $( (End of Larry Lesyna's mathbox.) $)
 
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Mathbox for Adhemar
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+
+  $( Replacement of a nested antecedent with an outer antecedent.  Commuted
+     simplificated form of elimination of a nested antecedent.  Also holds
+     intuitionistically.  Polish prefix notation:  CCCpqrCsCqr .  (Contributed
+     by ADH, 10-Nov-2023.)  (Proof modification is discouraged.) $)
+  adh-jarrsc $p |- ( ( ( ph -> ps ) -> ch ) -> ( th -> ( ps -> ch ) ) ) $=
+    ( wi jarr ax-1 ax-mp pm2.04 ) DABECEZBCEZEZEZJDKEELMABCFLDGHDJKIH $.
+  $( $j usage 'adh-jarrsc' avoids 'ax-3'; $)
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Minimal implicational calculus
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  Minimal implicational calculus, or intuitionistic implicational calculus, or
+  positive implicational calculus, is the implicational fragment of minimal
+  calculus (which is also the implicational fragment of intuitionistic calculus
+  and of positive calculus).  It is sometimes called "C-pure intuitionism"
+  since the letter C is used to denote implication in Polish prefix notation.
+  It can be axiomatized by the inference rule of modus ponens ~ ax-mp together
+  with the axioms { ~ ax-1 , ~ ax-2 } (sometimes written KS), or with
+  { ~ imim1 , ~ ax-1 , ~ pm2.43 } (written B'KW), or with { ~ imim2 ,
+  ~ pm2.04 , ~ ax-1 , ~ pm2.43 } (written BCKW), or with the single axiom
+  ~ adh-minim , or with the single axiom ~ adh-minimp .  This section proves
+  first ~ adh-minim from { ~ ax-1 , ~ ax-2 }, followed by the converse, due to
+  Ivo Thomas; and then it proves ~ adh-minimp from { ~ ax-1 , ~ ax-2 }, also
+  followed by the converse, also due to Ivo Thomas.
+
+  Sources for this section are
+  * Carew Arthur Meredith, _A single axiom of positive logic_, The Journal of
+    Computing Systems, volume 1, issue 3, July 1953, pages 169--170;
+  * Ivo Thomas, _On Meredith's sole positive axiom_, Notre Dame Journal of
+    Formal Logic, volume XV, number 3, July 1974, page 477, in which the
+    derivations of { ~ ax-1 , ~ ax-2 } from ~ adh-minim are shortened (compared
+    to Meredith's derivations in the aforementioned paper);
+  * Carew Arthur Meredith and Arthur Norman Prior, _Notes on the axiomatics of
+    the propositional calculus_, Notre Dame Journal of Formal Logic, volume IV,
+    number 3, July 1963, pages 171--187; and
+  * the webpage
+    ~ https://web.ics.purdue.edu/~~dulrich/C-pure-intuitionism-page.htm
+    on Dolph Edward "Ted" Ulrich's website, where these and other single
+    axioms for the minimal implicational calculus are listed.
```

Comment:
It would be nice to add these to our [Bibliography](https://us.metamath.org/mpeuni/mmset.html#bib) and then add bibliographic links following instructions given in the [conventions page](https://us.metamath.org/mpeuni/conventions.html#Bibliography%20references), under "Bibliography References".

Reply:
Oh I realize those lines were identical in @benjub's original chapter description. A formal bibliographic reference would still be a nice to have!

Reply:
If there is only one reference to a work, my understand is that it is optional whether to put it in the bibliography or inline. There's also a bit of a judgement call about whether to cite everything we found, or just a few key sources (often textbooks or the like). Issues include that some sources are paywalled, not online at all, or web pages which might not be around in a decade.

Not a blocker for merging this pull request, we can tweak this at our leisure if we want to.

Reply:
Indeed, I didn't add this reference to the bibliography because it is used only here, for a specialized purpose. I think it's probably better to leave it that way.